torch/onnx/_internal/exporter/_core.py

```diff
@@ -740,19 +891,34 @@ def _prepare_exported_program_for_export(
 
     graph_module = exported_program.graph_module
     # Include explicit type promotion nodes
-    graph_module = _fx_passes.insert_type_promotion_nodes(graph_module)
+    _fx_passes.insert_type_promotion_nodes(graph_module)
```

Comment:
I wonder why you changed the way to call this pass, would it be easier to have the same signature for all of them?